From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-hugetlb-move-hugetlb-cma-code-in-to-its-own-file-checkpatch-fixes
Date: Wed Jan 29 03:47:56 PM PST 2025

WARNING: added, moved or deleted file(s), does MAINTAINERS need updating?
#376: 
new file mode 100644

WARNING: Improper SPDX comment style for 'mm/hugetlb_cma.h', please use '/*' instead
#643: FILE: mm/hugetlb_cma.h:1:
+// SPDX-License-Identifier: GPL-2.0

WARNING: Missing or malformed SPDX-License-Identifier tag in line 1
#643: FILE: mm/hugetlb_cma.h:1:
+// SPDX-License-Identifier: GPL-2.0

total: 0 errors, 3 warnings, 481 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

./patches/mm-hugetlb-move-hugetlb-cma-code-in-to-its-own-file.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: Frank van der Linden <fvdl@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/hugetlb_cma.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/hugetlb_cma.h~mm-hugetlb-move-hugetlb-cma-code-in-to-its-own-file-checkpatch-fixes
+++ a/mm/hugetlb_cma.h
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: GPL-2.0
+/* SPDX-License-Identifier: GPL-2.0 */
 #ifndef _LINUX_HUGETLB_CMA_H
 #define _LINUX_HUGETLB_CMA_H
 
_
